code/modules/power/lighting.dm

```diff
@@ -132,18 +132,46 @@
 				for (var/dir in cardinal)
 					T = get_step(src,dir)
 					if (istype(T,/turf/simulated/wall) || (locate(/obj/wingrille_spawn) in T) || (locate(/obj/window) in T))
+						var/is_jen_wall = 0 // jen walls' ceilings are narrower, so let's move the lights a bit further inward!
+						if (istype(T, /turf/simulated/wall/auto/jen) || istype(T, /turf/simulated/wall/auto/reinforced/jen))
+							is_jen_wall = 1
 						src.dir = dir
 						if (dir == EAST)
-							src.pixel_x = 10
+							if (is_jen_wall)
+								src.pixel_x = 12
+							else
+								src.pixel_x = 10
 						else if (dir == WEST)
-							src.pixel_x = -10
+							if (is_jen_wall)
+								src.pixel_x = -12
+							else
+								src.pixel_x = -10
 						else if (dir == NORTH)
-							src.pixel_y = 21
+							if (is_jen_wall)
+								src.pixel_y = 24
+							else
+								src.pixel_y = 21
 						break
 				T = null
 
 
 
+//big standing lamps
+/obj/machinery/light/blamp
+	name = "big lamp"
+	icon = 'icons/obj/lighting.dmi'
+	desc = "A tall and thin lamp that rests comfortably on the floor."
+	anchored = 1
+	light_type = /obj/item/light/bulb
+	allowed_type = /obj/item/light/bulb
+	fitting = "bulb"
+	light_name = "light bulb"
+	brightness = 1.4
+	var/state
+	var/lampshaded = 1
```

Comment:
Are these vars used anywhere?

Reply:
Oh, no! Jeez, I forgot about this... This was something I started to do but decided against. Removing.

Reply:
Doesn't seem to be removed?

Reply:
NOW it's done! Oops, I could've sword I'd deleted it.